src/pip/_internal/models/link.py

```diff
@@ -248,6 +248,11 @@ def from_json(
         yanked_reason = file_data.get("yanked")
         dist_info_metadata = file_data.get("dist-info-metadata")
         hashes = file_data.get("hashes", {})
+        link_hash = None
+        if hashes:
+            for hash_name in _SUPPORTED_HASHES:
+                if hash_name in hashes:
+                    link_hash = LinkHash(name=hash_name, value=hashes[hash_name])
```

Comment:
```suggestion
                    link_hash = LinkHash(name=hash_name, value=hashes[hash_name])
                    break
```

Reply:
@cboylan a break is missing here, right?

Reply:
Yes, I'll push a new version shortly.